### ISSUE TITLE
Footer Items

### ISSUE THREAD

**Issue by phoxinsox** [tickety/gameofthecentury#3]
We are currently planning on having three similar forms for users to contact developers - support, feedback, and contact_us. These would be easy to do together - the only changes between pages would be small bits of wording. This can become its own mini-project as well, trying to set this up to drop us an email if someone uses the tool. 
Question - do we want three different types of forms, all of them essentially a return email and an explanation?

**Comment by tickety** [comment]
The same exact form, just a title difference?

**Comment by phoxinsox** [comment]
Yes, I believe so. It'd be return email, subject, message. We could make a dropdown for the subject as well.
